inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/DataNodeClient.java

```diff
@@ -107,4 +107,17 @@ public Boolean delete(Integer id) {
         return response.getData();
     }
 
+    /**
+     * Delete data node by name.
+     *
+     * @param name node name to be deleted
+     * @return whether succeed
+     */
+    public Boolean delete(String name) {
```

Comment:
Please specify the data node type at the same time.

Reply:
fixed, thx